packages/lesswrong/components/review/ReviewVoteTableRow.tsx

```diff
@@ -193,8 +197,16 @@ const ReviewVoteTableRow = (
                   {v}
               </LWTooltip>
             )}
+            
+          </div>
+          {eligibleToNominate(currentUser) && <div className={classes.yourVote}>
+            <PostsItemReviewVote post={post} marginRight={false}/>
+          </div>}
+          {currentUserIsAuthor && <LWTooltip title="You can't vote on your own posts">
+            <div className={classes.disabledVote}>Vote</div>
+          </LWTooltip>}
         </div>}
-        {getReviewPhase() !== "REVIEWS" && <div className={classes.votes}>
+        {getReviewPhase() !== "REVIEWS" && eligibleToNominate(currentUser) && <div className={classes.votes}>
```

Comment:
Because we are showing the /reviewVoting page to non-eligible voters, it's now necessary to hide the voting UI for them.

Reply:
Is eligible to nominate and eligible to vote different things?

Reply:
eligibleToNominate used to be used for nominations, then we got rid of nominations so there's only voting. It should be renamed, but I haven't had time to sit and do the streamline/refactoring/renaming pass.